src/DesktopHeader.jsx

```diff
@@ -99,6 +73,63 @@ class DesktopHeader extends React.Component {
     );
   }
 
+  renderUserMenu() {
+    return (getConfig().ENABLE_HEADER_WITHOUT_USERNAME ? this.userMenuWithoutUsername() : this.userMenuWithUsername());
+  }
+
+  // Renders an optional App Menu for
+  renderAppMenu() {
+    const { appMenu } = this.props;
+    const { content: appMenuContent, menuItems } = appMenu;
+    return (
+      <Menu transitionClassName="menu-dropdown" transitionTimeout={250}>
+        <MenuTrigger tag="a" className="nav-link d-inline-flex align-items-center">
+          {appMenuContent} <CaretIcon role="img" aria-hidden focusable="false" />
+        </MenuTrigger>
+        <MenuContent className="mb-0 dropdown-menu show dropdown-menu-right pin-right shadow py-2">
+          {menuItems.map(({ type, href, content }) => (
+            <a className={`dropdown-${type}`} key={`${type}-${content}`} href={href}>{content}</a>
+          ))}
+        </MenuContent>
+      </Menu>
+    );
+  }
+
+  renderMainMenu() {
```

Comment:
Please keep the same order as the original file.

Reply:
Can we try fixing the lint error instead of changing the complete file?

Reply:
The linter error was about the order of functions

Reply:
But let me try to fix the lining error without changing the order.